src/lib/components/modal/code/SaveNewCode.tsx

```diff
@@ -1,14 +1,14 @@
 import type { ButtonProps } from "@chakra-ui/react";
-import { Button, Icon, useToast, FormControl } from "@chakra-ui/react";
+import { Button, useToast, FormControl } from "@chakra-ui/react";
 import { useWallet } from "@cosmos-kit/react";
 import { useQuery } from "@tanstack/react-query";
 import { useEffect, useMemo, useState } from "react";
-import { MdBookmark, MdCheckCircle } from "react-icons/md";
 
 import { ActionModal } from "../ActionModal";
 import { useLCDEndpoint } from "lib/app-provider";
 import type { FormStatus } from "lib/components/forms";
 import { TextInput, NumberInput } from "lib/components/forms";
+import { CustomIcon } from "lib/components/icon/CustomIcon";
```

Comment:
After using `index.ts`, just import `lib/components/icon` and replace it everywhere.